stage1/enter_kvm/enter_kvm.go

```diff
@@ -63,9 +63,16 @@ func getPodDefaultIP(workDir string) (string, error) {
 	}
 	// kvm flavored container must have at first position default vm<->host network
```

Comment:
Argh, missed this. I'm no longer trying to base on position (nets[0] later is replaced with correct method of default network finding) so this comment should be removed...